package.json

```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "ember-cli-version-checker": "^1.2.0",
     "execa": "^0.6.0",
-    "fs-extra": "^2.0.0",
+    "fs-extra": "^3.0.0",
```

Comment:
I disagree and I would like to move to v3 because we can use the native promise returning functions then instead of having to promisify them manually